### DISCUSSION TITLE
Where to download https://github.com/rerdavies/pipedal/releases/download/v1.2.44/pipedal_1.2.44_arm64.deb

### DISCUSSION THREAD

**Discussion by maarthome** [rerdavies/pipedal#184] [Q&A]
Hi Robin,

Where can I download version 1.2.44?
(https://github.com/rerdavies/pipedal/releases/download/v1.2.44/pipedal_1.2.44_arm64.deb does not work...)

Cheers,
Maarten

**Comment by FoolHen** [comment]
The binaries are missing from the release. @rerdavies 😅

**Comment by maarthome** [comment]
I compiled the code according to Robin's description and got a working binary :-)
HALF the size of the previous ones.... 

Cheers

**Reply by rerdavies** [reply]
:-/ Binaries have been posted.
